packages/renderer/src/lib/ui/Checkbox.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+import Fa from 'svelte-fa/src/fa.svelte';
+import { faCheckSquare, faMinusSquare, faSquare } from '@fortawesome/free-solid-svg-icons';
+import { faSquare as faOutlineSquare } from '@fortawesome/free-regular-svg-icons';
+
+export let selected: boolean = false;
```

Comment:
curious: why use selected instead of checked ? As we provide the 'checked' property

Reply:
Fixed.